docs/csharp/tour-of-csharp/classes-and-objects.md

```diff
@@ -196,7 +196,7 @@ The previous four classes can be used to model arithmetic expressions. For examp
 
 [!code-csharp[ExpressionExample](../../../samples/snippets/csharp/tour/classes-and-objects/Program.cs#L40-L43)]
 
-The `Evaluate` method of an `Expression` instance is invoked to evaluate the given expression and produce a `double` value. The method takes as an argument a @`Dctionary` that contains variable names (as keys of the entries) and values (as values of the entries). The `Evaluate` method is a `virtual abstract` method, meaning that non-abstract derived classes must override it to provide an actual implementation.
+The `Evaluate` method of an `Expression` instance is invoked to evaluate the given expression and produce a `double` value. The method takes a `Dictionary` argument that contains variable names (as keys of the entries) and values (as values of the entries). The `Evaluate` method is an abstract method, and because abstract methods are implicitly virtual methods, non-abstract classes derived from `Expression` must override `Evaluate` and provide an actual implementation for the method.
```

Comment:
This now seems to imply that non-abstract classes must override all `virtual` methods, which is not true. I would remove the part about "virtual" altogether:

> The `Evaluate` method is an abstract method, which means that non-abstract classes derived from `Expression` must override `Evaluate` and provide an actual implementation for the method.